### DISCUSSION TITLE
Using Parcel in deployment

### DISCUSSION THREAD

**Discussion by ltc870** [parcel-bundler/parcel#8435] [Q&A]
Hello! I'm trying to deploy a website here on GitHub pages. The website was built with Parcel, so I'm not sure on how to deploy my website now. Also, the /dist folder is being ignored by git from. Would I have to unignore the folder for deployment? Any advice would be a big help!

**Comment by mischnic** [comment]
Yes, either you commit it or use something like https://github.com/tschaub/gh-pages#command-line-utility
So build it somewhere, and then `gh-pages -d dist` would upload the current contents of `dist` and update the deployment

**Reply by ltc870** [reply]
Thank you, I  will try it out!
